server/channels/app/platform/config.go

```diff
@@ -326,12 +328,18 @@ func (ps *PlatformService) LimitedClientConfig() map[string]string {
 }
 
 func (ps *PlatformService) IsFirstUserAccount() bool {
-	count, err := ps.Store.User().Count(model.UserCountOptions{IncludeDeleted: true})
-	if err != nil {
-		return false
+	if fetchUserCount {
+		count, err := ps.Store.User().Count(model.UserCountOptions{IncludeDeleted: true})
+		if err != nil {
+			return false
+		}
+		// Avoid calling the user count query in future if we get a count > 0
+		if count > 0 {
+			fetchUserCount = false
```

Comment:
Nit: might be clearer to `return false` here, `return true` below this if statement and then `return false` at the end of the function since while the value of `fetchUser` might always be right for what's returned that's not really what we're returning. 0/5 though

Reply:
I like this suggestion. 😄

Reply:
Done!